src/bin/electrs.rs

```diff
@@ -46,14 +49,15 @@ fn run_server(config: &Config) -> Result<()> {
             full_compaction(store)
         } else {
             // faster, but uses more memory
-            let store = bulk::index_blk_files(&daemon, config.bulk_index_threads, &metrics, store)?;
+            let store = bulk::index_blk_files(&daemon, &config, &metrics, store)?;
             let store = full_compaction(store);
             index.reload(&store); // make sure the block header index is up-to-date
             store
         }
     }.enable_compaction(); // enable auto compactions before starting incremental index updates.
 
-    let app = App::new(store, index, daemon)?;
+    let tx_cache = TransactionCache::new(config.tx_cache_size);
```

Comment:
The line above is not needed (following #2)

Reply:
Fixed.